Add warning notice for installations (#524)
OpenZeppelin/cairo-contracts: README.md

pip install openzeppelin-cairo-contracts
```

> ⚠️ Warning! ⚠️

> <br>

Reviewer: It would be good to add the actual warning inside the block? looks better for me to catch the user's attention with the warning explanation too.

Reviewer: <img width="902" alt="Screenshot 2022-11-25 at 13 01 00" src="https://user-images.githubusercontent.com/26070224/203982215-b7788721-ab17-41ce-99c2-f3fa33b83c67.png">

Author: Agreed! And that was the idea, but the preview in VSCode misled me. All fixed :)

> Installing directly the `main` branch may contain incomplete or breaking implementations, download [official releases](https://github.com/OpenZeppelin/cairo-contracts/releases/) only.

Reviewer: This line looks broken in the readme for some reason.

Reviewer: <img width="902" alt="Screenshot 2022-11-25 at 13 01 00" src="https://user-images.githubusercontent.com/26070224/203982342-6a0e2ed8-d18f-46d8-bb30-f0ddc47689df.png">

pip install openzeppelin-cairo-contracts
```

> ⚠️ Warning! ⚠️
<br>

> Installing directly through GitHub may contain incomplete or breaking implementations.
> While we aim not to introduce such changes, we still strongly recommend installing through [official releases](https://github.com/OpenZeppelin/cairo-contracts/releases/).

Reviewer 2: ```suggestion
Installing directly the `main` branch may contain incomplete or breaking implementations, download [official releases](https://github.com/OpenZeppelin/cairo-contracts/releases/) only.
```

this wording might be more accurate. installing through git, not just github, should also be discouraged
